src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs

```diff
@@ -89,7 +89,7 @@ public static string GetFullPath(string path, string basePath)
                 // Drive relative paths
                 Debug.Assert(length == 2 || !PathInternal.IsDirectorySeparator(path[2]));
 
-                if (GetVolumeName(path.AsSpan()).EqualsOrdinal(GetVolumeName(basePath.AsSpan())))
+                if (GetVolumeName(path.AsSpan()).EqualsOrdinalIgnoreCase(GetVolumeName(basePath.AsSpan())))
```

Comment:
There are other places in this file where `EqualsOrdinal` is used, can you change those as well?
https://github.com/dotnet/runtime/blob/7a60900cd1be3361773237d0d0d0293146db5547/src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs#L349-L353

Reply:
sure. done that. However, I omitted `EqualsOrdinal(@"\\".AsSpan()` as there is nothing to compare case-insensitive. If it should be consistent though, i can chance again of course.

Reply:
Are you sure this is correct for \\SERVER\SHARE as I think this is a valid volume name on Windows.

In that case the server name should be case insensitive, but the share name might not (Think non windows OSs like Samba on linux)

Reply:
`GetVolumeName` is only used when `path` is drive relative like C:foo, it's used to check whether `path` and `basePath` have the same root, the method is probably more complex than it needs to but I don't think it would affect UNC paths.

Reply:
Actually that was my thought when applying the changes :)